web/packages/selfhosted/test/polyfill/object_data/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>Object MIME insensitive</title>
+    </head>
+
+    <body>
+        <div id="test-container">
+            <object
+                data="uniwar.swf"
```

Comment:
Could you please change this to use the swf that we provide in the tests? We may later add "check if the swf actually loads and doesn't give an error" into the tests for extra security, and this would fail that